server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -520,6 +581,18 @@ export class MongoDbFactory implements core.IDbFactory {
 			global ? this.globalDbEndpoint : this.operationsDbEndpoint,
 			options,
 		);
+		for (const monitoringEvent of this.dbMonitoringEventsList) {
+			connection.on(monitoringEvent, (event) => {
+				// Using an event here so that we can use geneva monitoring in the future if we want to build alerts.
+				const eventWithName = { ...event, MonitoringEventName: monitoringEvent };
```

Comment:
what properties does the `event` object include? Is there any sensitive information there?

Reply:
Good call,  I think it should be fine based on existing examples:
1. https://www.mongodb.com/docs/drivers/node/v4.13/fundamentals/monitoring/cluster-monitoring/#serverdescriptionchanged
2. These events would have PII info, however I didn't monitor on these https://www.mongodb.com/docs/drivers/node/v4.13/fundamentals/monitoring/command-monitoring/#example-event-documents
3. https://www.mongodb.com/docs/drivers/node/v4.13/fundamentals/monitoring/connection-monitoring/#connectionpoolcreated